src/components/Navbar/Navbar.jsx

```diff
@@ -26,6 +33,14 @@ const Navbar = ({ isAdmin, changesMade }) => {
     { text: 'Emergency Numbers', path: '/emergency-numbers' },
```

Comment:
Can you fix this so that the path is just `/numbers`? And change it anywhere else where it's incorrect. Thank you :)